notebooks/data-sources/oc-github-repo/time_to_merge_model.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>A more convenient way of doing this is using pipelines. You could look at some experiments in the pipeline section I wrote for this <a href="https://gitlab.cee.redhat.com/shanand/kcs_candidates/-/blob/master/final_experiments.ipynb  " target="_blank">work</a>.</p><p>The benefit is that you can combine feature selection parameters and model search parameters and search for the best configuration. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
thanks! will check that out 🙂